packages/vitest/src/index.ts

```diff
@@ -36,6 +37,11 @@ export namespace Vitest {
     ): void
   }
 
+  /**
+   * @since 1.0.0
+   */
+  export type SchemaObj<A, E, R> = Array<Schema.Schema<A, E, R>> | { [K in string]: Schema.Schema<A, E, R> }
```

Comment:
Don't think you need any generics here. Also I don't think you can assign Schema.Never to this.

Reply:
Arbitrary doesn't support Schema.Never so I think not being able to assign Schema.Never to it is fine. But, yeah, this can be Schema.All w/o the generics

https://github.com/Effect-TS/effect/blob/b836fc429a9aad36e7a063557d1448ed95398c85/packages/schema/src/Arbitrary.ts#L154